app/models/planning_application.rb

```diff
@@ -476,6 +476,12 @@ def permitted_development_right
     permitted_development_rights.last
   end
 
+  def can_review_permitted_development_rights?
+    return awaiting_determination? unless recommendation
+
+    (awaiting_determination? || awaiting_correction?) && recommendation.challenged?
```

Comment:
Awaiting determination - that's when the assessor has finished their job and passed it to the reviewer?
Awaiting correction - ?
Recommendation.challanged - ?